src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/categories.yml

```diff
@@ -5,6 +5,8 @@ admin_categories_index:
         _controller: PrestaShopBundle:Admin\Sell\Catalog\Category:index
         _legacy_controller: AdminCategories
         _legacy_link: AdminCategories
+        _legacy_parameters:
+            id_category: categoryId
```

Comment:
@jolelievre this parameter will not become mandatory for other URL generation statements ?

Reply:
Good question! But I don't think so the converter only uses them to try and rename the parameters in their routing equivalent:
https://github.com/PrestaShop/PrestaShop/blob/7633be8294d90148d2bdfe3fd1c2d403e72ec91b/src/PrestaShopBundle/Routing/Converter/LegacyUrlConverter.php#L151

Nothing special happens if they are not present, or if any additional parameter is used. At worst they will be appended to the generated url

Reply:
Ok 👍 good to me then